kinoshita/pracitce-erd/instagram/laravel/database/migrations/2023_12_03_084141_create_comments_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('comments', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->bigInteger('commentable_id');
```

Comment:
基本、フォレーンキーはunsigned(正負なし)の方にしましょう
```suggestion
            $table->unsignedBigInteger('commentable_id');
```

Reply:
ちなみに、ポリモーフィック系において`morphs`というメソッドがあるので、それを使いましょう

https://readouble.com/laravel/10.x/ja/migrations.html#column-method-morphs

Reply:
こちらで修正しました

[d0fdebb](https://github.com/StoryEdutech/study_repo/pull/88/commits/d0fdebb9af00947921dd13c8988d5b653b0a8783)